pages/Documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/Regular/6.7.0.md

```diff
@@ -30,8 +30,8 @@ SDK will have the capability of clearing the welcome message as well including t
 
 When using the Control History API, conversations are filtered in real time based on the parameters provided, once each conversation dialog is closed the Conversation Screen will reflect the changes accordingly.
 
-{: .warning}
-To enable displaying the Welcome Message when using this feature, enable the following configuration `enableWelcomeMessageForHistoryControlAPI`, this is only supported when setting `LPConversationsHistoryStateToDisplay.Open`. To see more about how to achieve this visit [link]()
+{: .attn-alert}
```

Comment:
Midair collision—I had just refactored this and other classes, see #1931. Could you either leave this unchanged, or rename to “.attn-note”? This seemed more in line with what’s intended here.

Reply:
(Will handle this change on my end)